code/game/gamemodes/clock_cult/clock_scriptures/scripture_applications.dm

```diff
@@ -176,13 +176,13 @@
 /datum/clockwork_scripture/create_object/sigil_of_transmission
 	descname = "Structure Battery"
 	name = "Sigil of Transmission"
-	desc = "Scribes a sigil beneath the invoker which stores power to power clockwork structures."
+	desc = "Places a sigil that stores power to power clockwork structures."
```

Comment:
can it be "energy to power" rather than "power to power", that just sounds kind of blegh.